.github/workflows/prcheck.yml

```
@@ -33,4 +32,4 @@ jobs:
       - name: Check translations
         run: npm run i18n && npm run i18n:en
       - name: Build
-        run: npm run build
+        run: npm run build
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)